docs/dev-machine-setup.md

```diff
@@ -15,7 +15,56 @@ outlined in the [Getting Started](getting-started.md) instructions).
 
 ### Local Virtual Machine
 
-`[TODO]`
+Firecracker can also be installed in a Proxmox VM which uses KVM as Hypervisor. The nested option must be enabled on the host where Proxmox is running. In the following example nested virtualization has not been enabled yet ("N" No) so to check this in your Proxmox host excecute the following command:
+
+```
+cat /sys/module/kvm_intel/parameters/nested 
+N
+```
+
+To enable nested virtualization follow next steps:
+
+```
+echo "options kvm-intel nested=Y" > /etc/modprobe.d/kvm-intel.conf 
+
```

Comment:
Remove the trailing blank line from the code snippet. This also applies to lines  34, 29, 43, 52, 62 (double check in case I missed any).